### PR TITLE
Error messages

### PR DESCRIPTION
**Pull request by SeanMNorton**
This should be reviewed and merged after the **validation** branch.

### CODE REVIEW

File: config/cucumber.yml [hunk 1]
```diff
@@ -5,6 +5,6 @@ rerun_opts = rerun.to_s.strip.empty? ? "--format #{ENV['CUCUMBER_FORMAT'] || 'pr
 std_opts = "#{pretty} --tags 'not (@devready or @wip)'"
 %>
 default: <%= std_opts %> features
-wip: --tags @wip:3 --wip features
+wip: --tags @wip:5 --wip features
```
Comment (DocOnDev) [inline config/cucumber.yml]: WIP limit of 5 for 2 people? hmmm.....

Comment (SeanMNorton) [reply config/cucumber.yml]: Running the outline scenarios count as one for each username

Comment (SeanMNorton) [reply config/cucumber.yml]: We can change it back now that they aren’t in wip

Comment (DocOnDev) [reply config/cucumber.yml]: Ah. Just messing with you (mostly).

**Pull request merged**